backend/db/connectDb.ts

```diff
@@ -1,18 +1,15 @@
+import { config } from '../config'
 import mongoose from 'mongoose'
-import dotenv from 'dotenv'
-
-dotenv.config()
 
 let mongoURI: string
-if (process.env.MONGO_URI) {
-  mongoURI = process.env.MONGO_URI
+if (config.mongoURI) {
+  mongoURI = config.mongoURI
 } else {
```

Comment:
Unnnecessary validation, you already did that in the config file.

Reply:
you're right, removed this part!